programmers/문제 풀이/그래프/2468.py

```
@@ -0,0 +1,32 @@
+area = [list(map(int, input().split())) for _ in  range(N)]
+max_hegiht = max(max(row) for row in area)
+
+dx = [-1, 1, 0, 0]
+dy = [0, 0, -1, 1]
+
+def dfs(x, y, water_lever, visited):
+    visited[x][y] = True
+    for dir in range(4):
+        nx, ny = x + dx[dir], y + dy[dir]
+        if 0 <= nx < N and 0 <= ny < N:
+            if not visited[nx][ny] and area[nx][ny] > water_lever:
```

Ruff (0.12.2)

Use a single `if` statement instead of nested `if` statements

(SIM102)